documentation/asciidoc/computers/getting-started/setting-up.adoc

```diff
@@ -68,7 +68,7 @@ image::images/peripherals/sd-card.png[alt="Inserting a microSD card into a Raspb
 
 [[recommended-capacity]]
 
-We recommend using an SD card with at least 16GB of storage with Raspberry Pi OS. If you use Raspberry Pi OS Lite, we recommend at least 4GB.
+We recommend using an SD card with at least 16GB of storage for Raspberry Pi OS installations. For Raspberry Pi OS Lite, we recommend at least 4GB. You can use any SD card whose capacity exceeds this minimum size; there is no maximum size. As with any other boot media, you'll see improved performance on SD cards with faster read and write speeds.
```

Comment:
While strictly true regarding the maximum size, there's an implementation wrinkle we should probably make people aware of.

Today, the boot process expects an MBR partition table. MBR partition tables can support partitions up to 2TB in size. While I've yet to see a micoSD card larger than 1.5TB, we're close enough that it might be worth capturing that.

There are some workarounds for larger still drives, such as using a Hybrid GPT/MBR partitioning scheme: https://github.com/raspberrypi/firmware/issues/1465#issuecomment-814005398

However, directly using GPT partition tables is, to my understanding, not available today. So the naive way of using >2TB drives would not work. Paging @timg236 for visibility.

Reply:
4TB ... too rich for me!

The bootloader uses 32-bit logical block addresses so 2TB I think is the limit for now. No promises for when this will happen but I think NVMe would be the highest priority for very large drives and 4K native sectors.

Hybrid/GPT that have MBR / FAT at the start of the disk should work, but I'm fairly sure that A/B booting won't work unless all the FAT partitions appear in the "MBR"